core/src/test/java/org/chodavarapu/datamill/db/impl/QueryBuilderImplTest.java

```diff
@@ -211,21 +211,27 @@ public void deleteQueries() {
         TestQueryBuilderImpl queryBuilder = new TestQueryBuilderImpl();
 
         queryBuilder.deleteFrom("table_name").all();
-        assertEquals("DELETE FROM table_name", queryBuilder.getLastQuery());
+        assertEquals("DELETE  FROM table_name", queryBuilder.getLastQuery());
```

Comment:
The test here is right - it should only be one space - the code should result in one space